src/planner/aggregate_plan.go

```diff
@@ -16,6 +16,7 @@ import (
 
 	"github.com/xelabs/go-mysqlstack/sqlparser"
 	"github.com/xelabs/go-mysqlstack/xlog"
+	"strings"
```

Comment:
The system "strings" import should up the vendor import, like:
```
import (
	"encoding/json"
	"fmt"
	"strings"

	"github.com/pkg/errors"

	"github.com/xelabs/go-mysqlstack/sqlparser"
	"github.com/xelabs/go-mysqlstack/xlog"
)
```

Reply:
ok